lib/Imagine/Imagick/Image.php

```diff
@@ -696,8 +696,15 @@ private function applyImageOptions(\Imagick $image, array $options, $path)
             } else {
                 $compression += 5;
             }
-
-            $image->setImageCompressionQuality($compression);
+            $v = \Imagick::getVersion();
+            preg_match('/ImageMagick ([0-9]+\.[0-9]+\.[0-9]+)/', $v['versionString'], $v);
+            if (version_compare($v[1], '6.8.7') < 0 ) {
+                //Use this for ImageMagick releases before 6.8.7-5
+                $image->setImageCompressionQuality($compression);
+            } else {
+            //Use this for ImageMagick releases after 6.8.7-5
```

Comment:
Can we get the indentation right here? Also shouldn't there be a space after the //?